README.md

```diff
@@ -133,6 +134,31 @@ python analyze.py --config_file config.ini --log_level DEBUG
 
 See [Generic Command Line Parameters](#generic-command-line-parameters) for more details.
 
+# Analysis with sclite
+`sclite` is an open source tool designed to evaluate STT transcription results. More information about `sclite` can be found here -- https://people.csail.mit.edu/joe/sctk-1.2/doc/sclite.htm#sclite_name_0.
+
+This repo provides a wrapper script `analyze_with_sclite.py`, which can be used to run `sclite` and format its output.
```

Comment:
I suggest `optional_sclite_analyze.py` which plays nicer with `a<tab>` auto-completion

Reply:
Changed the script name to `optional_sclite_analyze.py`